core/utils.py

```diff
@@ -82,6 +84,12 @@ def attack(target, location, cookies=None, headers=None, payload=None, traverse=
     url = target+location
     print(colors("[~] Testing: {}".format(url), 93))
     try:
+        if dt:
+            res = requests.get(url, headers=headers, verify=False)
+            if res.status_code == 200:
+                return 1
```

Comment:
Since it is a loop wouldnt all other attack vectors run along with it if it didnt return from there? That was the reason I added it like that. Anyway changed like the requested way.